tests/integration/test_exec.py

```diff
@@ -1,12 +1,35 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
-from jina import Flow, Document
-from jinahub.SUB_PACKAGE_NAME.MODULE_NAME import MyDummyExecutor
+import os
+import librosa
 
+from jina import Flow, Document, DocumentArray
+from vggish import vggish_input
 
-def test_exec():
-    f = Flow().add(uses=MyDummyExecutor)
-    with f:
-        resp = f.post(on='/test', inputs=Document(), return_results=True)
-        assert resp is not None
+try:
+    from vggish_audio_encoder import VggishAudioEncoder
+except:
+    from jinahub.encoders.audio.vggish_audio_encoder import VggishAudioEncoder
```

Comment:
```suggestion
from jinahub.encoders.audio.vggish_audio_encoder import VggishAudioEncoder
```

Reply:
@bwanglzu if u do this, do the tests pass locally for u?

Reply:
They do not pass locally if I do this